vagrant_scripts/install_node.sh

```diff
@@ -16,7 +16,7 @@ nvm use
 nvm alias default current
 
 echo Update npm...
-npm install -g npm@2
+npm install -g npm@3
```

Comment:
Hmmm... I would have thought that using npm 3 would cause a lot of errors because of unmet peer dependencies. Is that not the case?